### DISCUSSION TITLE
Does ctrlr support M1 (edited)

### DISCUSSION THREAD

**Discussion by Godlike-Productions** [RomanKubiak/ctrlr#160] [Q&A]
I develop in Windows

Just wondering if any version of ctrlr OSX supports the A1 yet, and if so, which version?

Do I need to build the AU on an A1 Mac?

Just trying to plan the release of a new panel and want to support as many Mac's as possible.

**Comment by bijlevel** [comment]
Do you mean M1? (When I Google A1 Mac, I get directions the MacDonalds located on the highway A1:-))

**Reply by Godlike-Productions** [reply]
Yeah, the M1. D'Oh.

**Reply by bijlevel** [reply]
You can bet that there will be more and more M1 Macs coming this year, for example an iMac (Pro) as the next one. So wanting to know if Ctrlr is M1 proof is a good question. A soon as an M1 iMac (Pro) is on the market I will buy one and I can help you out. (But this answer doesn't help you at the moment with your question, sorry!)
